gridsome.config.js

```diff
@@ -65,7 +65,7 @@ module.exports = {
         feedItemOptions: node => ({
           title: node.title,
           description: node.description,
-          url: 'https://gridsome-starter-bleda.netlify.com/' + node.slug,
+          url: 'https://gridsome-starter-bleda.netlify.com/' + node.path,
```

Comment:
Remove trailing slash on URL, right now we're getting item URLs with a double slash, like this:

`https://gridsome-starter-bleda.netlify.com//writing-posts/`

This should work:

```
url: 'https://gridsome-starter-bleda.netlify.com' + node.path,
```

Reply:
Removed the trailing slash.